www/main/navigator/shabad/ShabadContent.jsx

```diff
@@ -148,6 +170,7 @@ const ShabadContent = () => {
     }
     setActiveVerse({ [verseIndex]: newTraversedVerse });
     if (activeVerseId !== newTraversedVerse) {
+      console.log('newTraversedVerse', newTraversedVerse);
```

Comment:
@theamanjs veerji, we can remove this console log here, I believe.

Reply:
fixed